addons/stock_landed_costs/models/stock_landed_cost.py

```diff
@@ -239,7 +239,7 @@ def compute_landed_cost(self):
 
                 former_cost = val_line_values.get('former_cost', 0.0)
                 # round this because former_cost on the valuation lines is also rounded
-                total_cost += tools.float_round(former_cost, precision_digits=digits) if digits else former_cost
+                total_cost += tools.float_round(former_cost, precision_rounding=rounding) if rounding else former_cost
```

Comment:
I think you should directly use the currency round method. I don't see a case where the company wouln't be set